src/wasm-lib/kcl/src/std/math.rs

```diff
@@ -239,7 +239,7 @@ pub async fn max(args: Args) -> Result<MemoryItem, KclError> {
     tags = ["math"],
 }]
 fn inner_max(args: Vec<f64>) -> f64 {
-    let mut max = std::f64::MAX;
+    let mut max = std::f64::MIN;
```

Comment:
Should `-infinity` also be considered?

Reply:
i cant imagine a part needing that and until it does this seems fine